content/vendor/guides/configuring-application-snapshots.md

```diff
@@ -0,0 +1,345 @@
+
+
+## Part I - Enabling Snaphots in KOTS
+
+
+These steps enable application snaphots by the vendor, which at a high level are:
+
+- [Add Backup Resource File](#add-backup-resource-file) - Adding this file pratcially enables snaphots in the application
+- [Configure Volumes](#configure-volumes) - Volumes are only part of the snapshot if they are annotated to be included.
+- [Configure Backup Hooks](#optionaladvanced-use-case-backup-hooks) - - May not be applicable to every situation, but these allow you to run commands prior to a backup.
+
+
+### Add Backup Resource File
+
+Create a new file named backup.yaml and add the following content:
+
+```shell
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: backup
+spec: {}
+
+```
+
+This is the minimal configuration and will suffice. 
+For further details on all of the available specs, please check the Velero documentation [here](https://velero.io/docs/v1.3.2/api-types/backup/)
+
+### Configure Volumes
+
+Volumes by default are not part of the snapshot.
+You configure which volumes to be part of the snaphot by adding the ```backup.velero.io/backup-volumes: ``` label annotation.
+
+In the AppDirect sample application, we have a PostgreSQL stateful set which has the following Volume:
+
+{{< notes title="Note" >}}
+This yaml is trucated for readability purposes. 
+The entire file is located [here](https://github.com/replicatedhq/kotsapps/blob/master/app-direct/manifests/postgres-deployment.yaml) in the GitHub repo
+{{< /notes >}}
+
+```shell
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: app-direct-postgresql
+  labels:
+    app: postgresql
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: postgresql
+  serviceName: postgres-service
+  template:
+    metadata:
+      labels:
+        app: postgresql   
+    spec:
+      containers:
+...
+      volumeMounts:
+        - name: postgresql-vct
+          mountPath: /var/lib/postgresql/data/pgdata
+          subPath: postgresql-db
+  volumeClaimTemplates:
+  - metadata:
+      name: postgresql-vct
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 2Gi
+
+```
+
+To have the volume included in the snapshot, we add the label annotation as shown below:
+
+```shell
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: app-direct-postgresql
+  labels:
+    app: postgresql
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: postgresql
+  serviceName: postgres-service
+  template:
+    metadata:
+      labels:
+        app: postgresql 
+      annotations:
+        backup.velero.io/backup-volumes: postgresql-vct # this volume will be included in snapshot
+    spec:
+      containers:
+...
+
+```
+
+The documentation also provides an example [here](https://kots.io/vendor/snapshots/configuring-backup/#identify-volumes) 
+
+### Optional/Advanced Use Case: Backup Hooks
```

Comment:
This is great content and should definitely be included somewhere, but maybe at the end or in a separate page. This is a tutorial, so we should not give users options, let's tell them exactly what to do. If this is optional, let's omit it or move it, if it's not optional for this tutorial then let's say so.